src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -28,7 +29,7 @@ export const ArtworkApp: React.SFC<Props> = props => {
   ]
 
   return (
-    <React.Fragment>
+    <HorizontalPadding px={[2, 4]}>
```

Comment:
Might be good to move this to `HorizontalPadding.defaultProps = { px: [2, 4] }` since most apps seem to follow these values and then later if things diverge too much we could pretty easily refactor.

Reply:
Updated this to go back to setting via `defaultProps`. 😅 Waffled a bit, I thought it was a little strange for a general padding component to set a default used by our route components when it could be used for horizontal padding in general, but I agree that it's widely used so makes senes to have in `defaultProps`, at least for now. Additionally, it saves us from having to do extra array checking before reaching into indices.